README.rst

```diff
@@ -1240,6 +1252,8 @@ Signals
 
 - ``instance``: instance of ``Device`` whose metrics have been received
 - ``request``: the HTTP request object
+- ``time``: time with which metrics will be saved. If none, then server time will be used
+- ``is_latest``: whether the data is latest or not
```

Comment:
The fact that use use `is_latest` in one place and `current` in the other does not convince me 100%.

Did you think about this?

Also, please let's clarify what this line means, a user may not understand what "whether the data is latest or not", I think we should write something like "whether the data has just been collected (latest data) or was collected previously and is being sent now due to network connectivity issues on the sender side".